src/js/tabs/receiver.js

```diff
@@ -631,6 +631,9 @@ TABS.receiver.initialize = function (callback) {
                 $('.tab-receiver .rcSmoothing-auto-factor').hide();
             }
 
+            const tooltip = i18n.getMessage(semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44) ? "receiverRcSmoothingAutoFactorHelp2" : "receiverRcSmoothingAutoFactorHelp");
+            $('.receiverRcSmoothingAutoFactorHelp').attr('title', tooltip);
```

Comment:
No need to attr here the tootltip for version < 1.44 since it is already defined and named in the html part.

Reply:
Fixed

Reply:
Now changed the values to be defaults using version control to address translation work

Reply:
The other numbers, where we mention higher values, won't be correct for 4.2.
I think it is better to have entirely separate tooltips for 4.2 and 4.3.
These situations arise from time to time.
We shouldn't change the 4.2 tooltip.
We probably have a similar issue with antigravity gain. In 4.2 it could be pushed to 9 or 10 and would only increase I. In 4.3 it increases P also, and has inherently higher gain, and typically should NOT be pushed to 9 or 10. I haven't had time to check all these tooltips but we will need separate tooltips, or a tooltip that explains "in 4.2, do X, in 4.3 do Y".

Reply:
We are free to add some extra defaults for those values too.

Reply:
This need a lotof maintain @ctzsnooze , we should use generic tooltips for all versions

Reply:
I agree, I think it would be simpler to just have the text for the tooltip.  I don't think we need the code that pulls the default value by version.  Just change the tooltip, if needed, by version.  
If we think about antigravity, say, at one version we make a big change, then for many versions after, most likely no further change.
But when we do make a change, it is not just the default value that changes, but perhaps the way the code works, or the tuning information.
So my feeling is that we should keep tooltips as simple as possible, but version control them so that when they change significantly, we make a new tooltip for the newer version.
We can't have the same tooltip for all versions, it would just be useless in many settings.  Eg transition.  Now we have jitter reduction we don't recommend transition anymore, so we should change that tooltip to explain the new situation in 4.3, and that tooltip would be different from the tooltip for transition before 4.3